packages/@react-stately/color/src/useColorAreaState.ts

```diff
@@ -135,11 +137,15 @@ export function useColorAreaState(props: ColorAreaProps): ColorAreaState {
     valueRef.current = color.withChannelValue(channels.yChannel, v);
     setColor(valueRef.current);
   };
+  let xChannelPageStep = Math.max(color.getChannelRange(channels.xChannel).pageSize, xChannelStep);
+  let yChannelPageStep = Math.max(color.getChannelRange(channels.yChannel).pageSize, yChannelStep);
```

Comment:
nit: can we avoid using the term "page" for this? It doesn't really make sense. There are no pages 😄

Reply:
yeah, happy to, got a better name?
bigStep?
leap?